app/views/reviews/edit.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row">
+
+  <div class="large-12">
+    <h1>Edit Review</h1>
+    <%= form_for [ @restaurant, @review ] do |f| %>
+      <%= f.label "value" %>
+      <%= f.select :value, Review::VALUES, :include_blank => true %>
+
+      <%= f.label "proximity" %>
+      <%= f.select :proximity, Review::PROXIMITY, :include_blank => true %>
+
+      <%= f.label "volume" %>
+      <%= f.select :volume, Review::VOLUME, :include_blank => true %>
+
+      <%= f.label "originality" %>
+      <%= f.select :originality, Review::ORIGINALITY, :include_blank => true %>
+
+      <%= f.label "body" %>
+      <%= f.text_area :body %>
+
+      <%= f.submit "Submit" %>
+    <% end %>
+  </div>
+</div>
```

Comment:
This whole form should also go into a partial! (The same one as shown on the Show page!)